sdk/src/beta9/abstractions/base/runner.py

```diff
@@ -130,6 +132,12 @@ def __init__(
         )
         self.extra: dict = {}
 
+        if (self.gpu != "" or len(self.gpu) > 0) and self.gpu_count == 0:
+            self.gpu_count = 1
+
+        if self.gpu_count > 2:
+            raise ValueError("GPU count cannot be greater than 2")
```

Comment:
I think this would be better on the server side - potentially under the stubLimits config - to prevent anyone from overriding this client side.